src/software/jetson_nano/redis/redis_client_test.cpp

```diff
@@ -9,6 +9,9 @@
 #include "cpp_redis/core/subscriber.hpp"
 #include "software/test_util/test_util.h"
 
+// TODO #2805: Convert this into a test fixture so that we can start and run a REDIS
+// server during the lifetime of the test fixture. Then re-enable all tests
```

Comment:
Is there a reason why they can't remain enabled until this task is done?

Reply:
because we need to have a redis server up and running for this to work

Reply:
Because these tests assume that there's a redis server running to GET and SET on

Reply:
It's doable for me to set this up via a test fixture but I figured it wasn't a high priority thing and we can do it later

Reply:
Okay sounds good